.github/workflows/python-publish.yml

```diff
@@ -91,3 +91,5 @@ jobs:
           repository-url: https://test.pypi.org/legacy/
       - name: Publish distribution 📦 to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          attestations: false
```

Comment:
this is a bad idea. at least move it to testpypi

Reply:
I had to set attestations as false as it was failing for release to pypi. But can you elaborate on your comment a bit more, as to why its not a good idea?